include/quicly.h

```diff
@@ -620,6 +620,10 @@ struct st_quicly_stream_t {
          * size of the receive window
          */
         uint32_t window;
+        /**
+         * maximum number of ranges (i.e. gaps + 1) permitted in `recvstate.ranges`
```

Comment:
As discussed, this might be a good place to document the logic for how this value is computed from the flow control window.